scss/components/_table-styles.scss

```diff
@@ -253,6 +253,84 @@ h3 + .simple-table {
   td:first-child {
     border-right: 1px solid $gray-medium;
     padding-left: u(4rem);
-    font-style: italic;
+     font-style: italic;
   }
 }
+
+// For custom_table_block in streamfactory
+
+.custom-table {
+  box-shadow: -8px 0px 9px -2px $gray-medium inset;
```

Comment:
Yes, thats a better nuance of the box-shadow settings for this that makes the shadow more subtle and hides the top/bottom shadow better.